File: app/src/main/java/Entry.java
import io.realm.RealmObject;

/* This is the Realm Entry Objects, basically the data model schema */
public class Entry extends RealmObject {
    private String id;
    private String title;
    private String userName;
    private String password;
    private String url;
    private String notes;

    public Entry(String id){
        this.id = id;
        userName = "";
        password = "";
        url = "";
        notes = "";
    }

    public String getTitle(){
        return title;
    }
    public String getUserName(){
        return userName;
    }
    public String getId(){
        return id;
    }
    public String getPassword(){
        return password;
    }
    public String getUrl(){
        return url;
    }
    public String getNotes(){
        return notes;
    }

    public void setTitle(String title){
        this.title = title;
    }
    public void setUserName(String userName){
        this.userName = userName;
    }
    public void setId(String id){
        this.id = id;
    }
    public void setPassword(String password){
        this.password = password;
    }
    public void setUrl(String url){
        this.url = url;
    }
    public void setNotes(String notes){
        this.notes = notes;
    }
}

File: app/src/main/java/xyz/willnwalker/yetanotherpasswordmanager/Entry.kt
package xyz.willnwalker.yetanotherpasswordmanager

import io.realm.RealmModel
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass
import java.util.*

/* This is the Realm xyz.willnwalker.yetanotherpasswordmanager.Entry Objects, basically the data model schema */
@RealmClass
open class Entry : RealmModel {
    @PrimaryKey
    var id = UUID.randomUUID().toString()
    var title = ""
    var userName = ""
    var password = ""
    var url = ""
    var notes = ""
}